gcloud/bigquery/client.py

```diff
@@ -102,20 +102,18 @@ def dataset(self, dataset_name):
         """
         return Dataset(dataset_name, client=self)
 
-    def _job_from_resource(self, resource):
+    def job_from_resource(self, resource):
```

Comment:
Yeah if we need public access, then I totally get it. I just wasn't sure where that happened.